src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorHandlerImpl.java

```diff
@@ -102,21 +116,35 @@ public void onNodeDragEnd(final NodeDragEndEvent event) {
 
     @Override
     public void onNodeMouseClick(final NodeMouseClickEvent event) {
-        if (clickTimer.isRunning()) {
-            clickTimer.cancel();
+        checkRunningTimer(clickTimer);
+        checkRunningTimer(mouseDownTimer);
```

Comment:
I guess there is no need for checking/stopping the scheduled tasks (timers), I would just call the `Timer.cancel()` method.
Also because I guess that lienzo fires the mouse down and click events almost consequentially, which could imply that the timer for mouse down is still not running (scheduled to `150ms`), so not even stopped, which means that will fire anyway as with the click one, right? (I guess what you want to achieve is only firing on of both timers)